T/tree-sitter-c/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "tree-sitter-c"
+version = v"0.16.0"
+
+# Collection of sources required to complete build
+sources = [
+    FileSource("https://github.com/tree-sitter/tree-sitter.git", "d8c3f472d23ad79f519651d5cf715b56467d35d0")
+    FileSource("https://github.com/tree-sitter/tree-sitter-c.git", "6002fcd5e86bb1e8670157bb008b97dbaf656d95")
```

Comment:
Uh, it looks like I got the types wrong :eyes: 

This should be:
```julia
   GitSource("https://github.com/tree-sitter/tree-sitter.git", "d8c3f472d23ad79f519651d5cf715b56467d35d0"),
   GitSource("https://github.com/tree-sitter/tree-sitter-c.git", "6002fcd5e86bb1e8670157bb008b97dbaf656d95")
```

Reply:
Gah! My tired eyes couldn't find the bug! thanks!

Reply:
> Gah! My tired eyes couldn't find the bug! thanks!

No worries!  Instead, thanks for testing the version in `master` and triggering the bug!